odoo/tests/addons/test_translation_import/tests/test_term_count.py

```diff
@@ -10,6 +10,21 @@ def test_count_term(self):
         Just make sure we have as many translation entries as we wanted.
         """
         odoo.tools.trans_load(self.cr, 'test_translation_import/i18n/fr.po', 'fr_FR', verbose=False)
-        ids = self.registry('ir.translation').search(self.cr, self.uid,
+        ids = self.env['ir.translation'].search(
             [('src', '=', '1XBUO5PUYH2RYZSA1FTLRYS8SPCNU1UYXMEYMM25ASV7JC2KTJZQESZYRV9L8CGB')])
         self.assertEqual(len(ids), 2)
+
+    def test_noupdate(self):
+        """
+        Make sure no update do not overwrite translations
+        """
+        menu = self.env.ref('test_translation_import.menu_test_translation_import')
+        menu.name = "New Name"
+        menu.with_context(lang='fr_FR').name = "Nouveau nom"
+        # reload with overwrite
+        odoo.tools.trans_load(self.cr, 'test_translation_import/i18n/fr.po', 'fr_FR', verbose=False, context={'overwrite': True})
+
+        # invalidate cache
+        menu.refresh()
```

Comment:
That's right:
- the `ormcache` is cleared by https://github.com/odoo/odoo/blob/10.0/odoo/tools/translate.py#L1142
- the record cache is not cleared automatically, hence the necessity to `refresh()`